lib/evaluate/intrinsics.cc

```diff
@@ -714,10 +714,24 @@ static const IntrinsicInterface genericIntrinsicFunction[]{
 // Tables 16.2 and 16.3 in Fortran 2018.  The "unrestricted" functions
 // in Table 16.2 can be used as actual arguments, PROCEDURE() interfaces,
 // and procedure pointer targets.
+// Note that the restricted conversion functions dcmplx, dreal, float, idint,
+// ifix and sngl are extended to accept any argument kind because this a common
```

Comment:
This line has more than 80 characters.

Reply:
As far as I can see it has 79 col in my editors (vim & notepad++). Plus clang-format is not breaking these line, and it should if they were over 80. Are you saying this because Github viewer is breaking the line on you screen, or is it more than 80 in you editor ?
(For the other lines you are pointing out to I also measure <= 80 col)

Reply:
My bad.  I was fooled by Windows 10 automatically resizing my terminal window.  Please ignore these line length comments.  I'll be more careful in the future.